### PR TITLE
[Eagle 952] system dashbord

### PR DESCRIPTION
**Pull request by koone**
https://issues.apache.org/jira/browse/EAGLE-952

### CODE REVIEW

**Comment by haoch** [comment]
@koone could you please add a screenshot about the feature in jira ticket?

File: eagle-metric/eagle-system-web/src/main/webapp/app/apps/system/partials/serverList.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+		<div id="serverList" sort-table="serverList" is-sorting="isSorting"
+			 search-path-list="searchPathList"
+			 scope="tableScope">
+			<table class="table table-bordered">
+				<thead>
+				<tr>
+					<th sortpath="tags.hostname">Host Name</th>
+					<th sortpath="tags.rack">Rack</th>
+					<th sortpath="tags.site">SiteId</th>
+					<th sortpath="status">Status</th>
```
Comment (chitin) [inline eagle-metric/eagle-system-web/src/main/webapp/app/apps/system/partials/serverList.html]: sortpath should be unique, and these element is considered to add to searchPathList.

**Comment by haoch** [comment]
@koone @chitin how about this PR?

**Comment by koone** [comment]
@haoch , i want add a system widget, so i will add a commit with widget and fix some problem today

**Comment by haoch** [comment]
@koone cool, please update once it's done.

**Comment by zombieJ** [comment]
Since stlye like `hadoopMetric-chart` is used in other application. You can pop it out as common style in `eagle-server`. But it's OK for current status.

**Pull request closed without merging**